### ISSUE TITLE
More detailed failure report

### ISSUE THREAD

**Issue by ericmasiello** [mfrachet/cypress-audit#191]
**What would you like to see in the library?**
I am using `cypress-audit` purely to test for accessibility failures. As such, my tests are very simple, e.g.

```ts
describe('Mobile Dock', () => {
    beforeEach(() => {
        cy.visit('/mobile-dock');
    });

    it('should pass lighthouse audit', () => {
        cy.lighthouse({
            accessibility: 100,
        });
    });

    it('should pass pa11y audit', () => {
        cy.pa11y();
    });
});
```

However, if i my lighthouse score fails, Cypress only reports that it did not meet the `100` value. What I'd like to know is which specific aspects of accessibility it failed on. 

**Why?**
This reduces friction in diagnosing accessibility problems. Right now, my only recourse is to manually rerun the same page in Chrome with Lighthouse and look its detailed output to understand why the failure occurred.

**Any other information that may help understanding the need**
N/A
